p2p/pex/pex_reactor.go

```diff
@@ -529,7 +538,7 @@ func (r *Reactor) dialPeer(addr *p2p.NetAddress) error {
 		// failed to connect to. Then we can clean up attemptsToDial, which acts as
```

Comment:
can you remove this comment?

I think we need both attemptsToDial and blacklist